drupal-http/hooks/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Start the Nginx server, passing it our bundled configuration file.
+exec {{ pkgPathFor "core/nginx" }}/bin/nginx -c "{{ pkg.svc_config_path }}/nginx.conf" 2>&1
```

Comment:
I am not sure I understand why you need `{{ pkgPathFor "core/nginx" }}/bin/nginx`. Is it not the same thing to use `nginx`?

Reply:
That's a good point, as long as some `nginx` is in the `pkg_deps` than `nginx` should be in `PATH` for the `run` hook and we can avoid this code dying when someone switches to a different nginx build